metaflow/plugins/kfp/kfp_step_decorator.py

```diff
@@ -0,0 +1,36 @@
+from metaflow.decorators import StepDecorator
+from metaflow.exception import MetaflowException
+
+
+class KfpStepDecorator(StepDecorator):
+    """
+    For KFP orchestrator plugin only.
+
+    Step decorator to specify the image for the step.
+
+    To use, follow the example below.
+    ```
+    @kfp_step(image="image_on_dockerhub")
+    @step
+    def start(self):
+        self.next(self.next_step)
+
+    ```
+
+    Parameters
+    ----------
+    image: str
+        Defaults to None.
+        Must resolve to an actual image, either publicly hosted
+        or available for download on the customer's infra.
+    """
+
+    name = 'kfp_step'
+
+    defaults = {
+        "image": None,
+    }
+
+    def step_init(self, flow, graph, step, decos, environment, datastore, logger):
+        if not self.attributes["image"]:
+            raise MetaflowException("You must specify the image.")
```

Comment:
I don't believe that `image=` is required... if not specified it's the decorator is ignored.

Reply:
Good point, will make that change.